foodalert/static/foodalert/js/components/generic-page.vue

```diff
@@ -63,6 +64,7 @@ export default {
   data() {
     return {
       notificationState: false,
+      logoutURL: logoutUrl,
```

Comment:
I think you may need to use 'this.logoutUrl" here

Reply:
Or, since logoutUrl is a global variable, I don't think this assignment is necessary

Reply:
logout URL also needs to be added to line 17